src/redux/books/RemoveBookBtn.js

```diff
@@ -0,0 +1,21 @@
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { removeBook } from 'redux/books/bookSlice';
+
+const RemoveBookBtn = ({ id }) => {
+  const dispatch = useDispatch();
+  const handleRemoveBook = () => dispatch(removeBook(id));
+  return (
+    <button
+      onClick={handleRemoveBook}
+      type="button"
+    >
+      remove
+    </button>
+  );
+};
+export default RemoveBookBtn;
+
+RemoveBookBtn.propTypes = {
+  id: PropTypes.string.isRequired,
+};
```

Comment:
- [ ] Please makes sure this component is in the `src/components` directory since it is the directory dedicated to contain all your app components :pray: